boards/PDM/Inc/Io/Io_Efuse_22XS4200.h

```diff
@@ -0,0 +1,646 @@
+/**
+ * @brief Driver for interfacing with the 22XS4200 E-Fuse via SPI
+ * @Settings Efuse max SPI transfer rate: 8MHz
+ *           SPI Clock Polarity: 0 (SCK low-level idle state)
+ *           SPI Clock Phase: 2nd edge (slave samples MOSI on SCK falling edge)
+ *           Slave Select: Active Low (Must be toggled between SPI messages)
+ */
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+#include "main.h"
+
+/* Current Sense Ratios */
+#define HIGH_CURRENT_SENSE_RATIO (1.0f / 1500.0f)
+#define LOW_CURRENT_SENSE_RATIO (1.0f / 500.0f)
+
+/* Current Sense Channels */
+#define AUX1_CURRENT_SENSE_CHANNEL CSNS_FUNCTION_CURRENT_CH0
```

Comment:
Are all the defines in this file necessary to be made public (i.e. placed in header)? If there's anything that can be made private it would be nice to leave them in `.c`

Reply:
Moved the ones that I could